content/docs/ops/volume-service-requests.md

```diff
@@ -0,0 +1,19 @@
+---
+menu:
+  docs:
+    parent: tenants
+layout: ops
+title: Handling volume service requests
+---
+
+Customers occasionally request that a volume of a certain size be created for use via volume services in their space. Here's how to handle those requests.
+
+1. Create a volume with the requested size and a hard-to-guess (ie random long) string for the name
+  1. (procedure here)
+2. Enable the nfs volume service broker in the requested organization 
+  1. (procedure here)
+3. [Enable the `volume-service-testing` isolation segment for their requested organization](https://docs.cloudfoundry.org/adminguide/isolation-segments.html#relationships)
```

Comment:
We might want to verify that nfs issues can actually affect unrelated apps on the cell before committing to a new isolation segment. Or we can just set it up and skip the research--it's not hard.

cc @sharms

Reply:
I'd rather just do it to ensure this can't b0rk other apps. As @sharms would say, we'll spend more and incur more customer sadness debugging problems than if we just go ahead and eliminate the possibility of there being any interference.

Reply:
Besides which, I don't think we'd put much effort into fixing problems given this is a stopgap.